src/helper.rs

```diff
@@ -1,3 +1,12 @@
+use reqwest::Url;
+use serde::{Serializer, Serialize};
+
 pub fn bool_to_u8(b: bool) -> u8 {
     if b { 1u8 } else { 0u8 }
 }
+
+pub fn serialize_uri<S>(uri: &Option<Url>, ser: S) -> Result<S::Ok, S::Error>
```

Comment:
Trying to understand the rationale for this. Did you add this to avoid using the `url_serde` crate?

Reply:
I don't remember exactly, it was something weird going on between `reqwest` and `url_serde` making it not compile.

Reply:
Interesting... I'll take a look at this.

Reply:
I reverted this change. I didn't run into any issues keeping `url_serde` around. Better to have the logic there than in this crate.